packages/utilities/src/object.ts

```diff
@@ -26,6 +28,7 @@ function checkProperties(a: any, b: any): boolean {
  *
  * @public
  */
+// tslint:disable-next-line:no-any
 export function shallowCompare(a: any, b: any): boolean {
```

Comment:
can't we change it to `export function shallowCompare<TA, TB>(a: TA, b: TB): boolean`?